src/middleware/packages/crypto/keys/keys-container.js

```
@@ -0,0 +1,51 @@
+const { ControlledContainerMixin } = require('@semapps/ldp');
+const { Errors: E } = require('moleculer-web');
+const KEY_TYPES = require('./keyTypes');
+
+/**
+ * DANGER ZONE
+ *
+ * Container to store the private keys of actors.
+ *
+ * Watch out with permissions. This should be strictly limited to the owner and privileged apps.
+ * @type {import('moleculer').ServiceSchema}
+ */
+module.exports = {
+  name: 'keys.container',
+  mixins: [ControlledContainerMixin],
+  settings: {
+    path: '/key',
+    acceptedTypes: Object.values(KEY_TYPES),
+    permissions: (webId, ctx) => {
+      // If not a pod provider, the container is shared, so any user can append.
+      return {
+        anyUser: {
+          // `this` is not available here.
+          read: !ctx.service.settings.podProvider,
+          append: !ctx.service.settings.podProvider
+        }
+      };
+    },
+    newResourcesPermissions: webId => ({
+      user: {
+        uri: webId,
+        read: true,
+        write: true,
+        control: true
+      }
+    }),
```

To improve security and prevent mistakes, we could throw an error if `webId` is "anon" or "system".